dev-scripts/directory-structure-maker/directory-structure-maker.sh

```diff
@@ -58,7 +65,7 @@ echo "Prepareing Choreo Connect folder structure..."
 docker cp $ADAPTER_IMAGE_ID:home/wso2/adapter           ./ADAPTER/\$HOME
 mv ./ADAPTER/\$HOME/adapter                             ./ADAPTER/\$HOME/adapter-$ARC_VALUE
 docker cp $ADAPTER_IMAGE_ID:home/wso2/check_health.sh   ./ADAPTER/\$HOME
-docker cp $ADAPTER_IMAGE_ID:home/wso2/LICENSE.txt       ./ADAPTER/\$HOME
+cp ../temp/LICENSE.txt                                     ./ADAPTER/\$HOME
```

Comment:
Can we align theses lines ? (Align with `./ADAPTER/\$HOME`)

Reply:
Done with https://github.com/wso2/product-microgateway/pull/3314/commits/3ba6bfd05424da4aea39fc3e02a68b6fb1967f65